neo/io/blackrockio_v4.py

```diff
@@ -1984,7 +1984,7 @@ def __read_channelindex(
             file_origin=self.filename)
 
         if index is not None:
-            chidx.index = index
+            chidx.index = np.array(index)
```

Comment:
Maybe for consistency (see base object):
```suggestion
            chidx.index = np.array(index, np.dtype('i'))
```